src/main/java/org/sonarsource/scanner/ant/SonarQubeTask.java

```diff
@@ -48,13 +57,37 @@ public void execute() {
 
   // VisibleForTesting
   void launchAnalysis(Properties properties) {
-    EmbeddedRunner.create()
-      .addProperties(properties)
+    EmbeddedRunner runner = EmbeddedRunner.create(new LogOutputImplementation())
+      .addGlobalProperties(properties)
       .unmask("org.apache.tools.ant")
       .unmask("org.sonar.ant")
-      .setApp("Ant", SonarQubeTaskUtils.getTaskVersion())
-      .addExtensions(getProject())
-      .execute();
+      .setApp("Ant", SonarQubeTaskUtils.getTaskVersion());
+
+    runner.start();
+    try {
+      runner.addExtensions(getProject());
+    } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1166)